ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/ScanResultsOverviewTable.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import {
+    TableComposable,
+    Tbody,
+    Td,
+    Th,
+    Thead,
+    Tr,
+    ActionsColumn,
+    IAction,
+} from '@patternfly/react-table';
+import { format } from 'date-fns';
+
+import usePagination from 'hooks/patternfly/usePagination';
+import useTableSort from 'hooks/patternfly/useTableSort';
+
+import { ComplianceScanResultsOverview } from './types';
+import ScanResultsToolbar, { SearchFilter } from './ScanResultsToolbar';
+
+import scanResultsOverviewMockData from './MockData/complianceScanResultsOverview.json';
+
+const defaultSearchFilter = {
+    scanName: '',
+    clusterName: '',
+    profileName: '',
+};
+
+const sortFields = ['Scan Name', 'Failing Controls', 'Last Scanned'];
+const defaultSortOption = { field: 'Last Scanned', direction: 'asc' } as const;
+
+function ScanResultsOverviewTable() {
+    const { page, perPage, onSetPage, onPerPageSelect } = usePagination();
+    const { sortOption, getSortParams } = useTableSort({ sortFields, defaultSortOption });
+    const [searchFilter, setSearchFilter] = useState<SearchFilter>(defaultSearchFilter);
+    const [scanResultsOverviewData] = useState<ComplianceScanResultsOverview[]>(
+        scanResultsOverviewMockData.scanOverviews
+    );
+
+    useEffect(() => {
+        // TODO: fetch GetComplianceScanResultsOverview once implemented and update scanResultsOverviewData
+    }, [searchFilter, page, perPage, sortOption]);
+
+    const defaultActions = (): IAction[] => [
+        {
+            title: 'Edit schedule',
+            // TODO: link to edit schedule page
+        },
+    ];
+
+    const displayOnlyItemOrItemCount = (items: string[], multipleItemLabel: string): string => {
+        if (items.length > 1) {
+            return `${items.length} ${multipleItemLabel}`;
+        }
+        return items[0];
+    };
```

Comment:
This looks like something we've done before. There should be a util function we could import, or this would be a good opportunity to make one.

Reply:
I'll update this in a later branch